app/domain/logs.rb

```diff
@@ -351,42 +351,42 @@ module AuthnJwt
       )
 
       AddingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
-        msg: "Adding JWT claim '{0-claim-name}' to list of mandatory claims to be validated",
+        msg: "Adding JWT claim, '{0-claim-name}' to list of mandatory claims to be validated...",
```

Comment:
missing ,
Adding JWT claim, '{0-claim-name}'**,** to list of mandatory claims to be validated..."

Reply:
Done. Strange its not pushed

Reply:
@tzheleznyak  can you provide example?